src/api/time/TimeContext.js

```diff
@@ -362,6 +362,19 @@ class TimeContext extends EventEmitter {
         this.boundsVal = newBounds;
         this.emit('bounds', this.boundsVal, true);
     }
+
+    /**
+     * Checks if this time context is in real-time mode or not.
+     * @returns {boolean} true if this context is in real-time mode, false if not
+     * @private
+    */
+    isRealtime() {
+        if (this.clock()) {
```

Comment:
should we add a quick test in `TimeAPISpec` to cover this new method?

Reply:
The e2e test in the YAMCS plugin is testing this implicitly, but I agree a test in the core openmct is a good idea.